tools/mc_ledger_test.py

```diff
@@ -0,0 +1,76 @@
+import local_network
+import time
+
+from test_utils.utilities import parse_network_type_cmd_line_args
+from test_utils import fullservice as fslib
+
+# run sample test transactions between the first two accounts in full service
+def test_transactions(fs):
+    print(('________________________________________________________________________________'))
+    print('testing transaction sends')
+    if fs.account_ids is None:
+        print(f'accounts not found in wallet')
+        local_network.cleanup(1)
+    elif len(fs.account_ids) < 2:
+        print(f'found {len(fs.account_ids)} account(s), minimum required is 2')
+        local_network.cleanup(1)
+    account_0 = fs.account_map[fs.account_ids[0]]
+    account_1 = fs.account_map[fs.account_ids[1]]
+    p_mob_amount = str(600_000_000)
+
+    # flakey tests due to accounts having a variable amount of pmob. This needs to be controlled for use.
+    log_0 = fs.send_transaction(account_0['account_id'], account_1['main_address'], p_mob_amount)
```

Comment:
Potential improvement: A balance check before the test could make it easier to understand why the test is failing, if it fails